demos/benchmark/lv_demo_benchmark.c

```diff
@@ -490,9 +584,64 @@ static uint32_t rnd_act;
  *   GLOBAL FUNCTIONS
  **********************/
 
+static void benchmark_context_init(benchmark_context_t * context)
+{
+#if LV_USE_FREETYPE
+    context->font_bitmap = lv_freetype_font_create(LV_TEST_FONT_PATH,
+                                                   LV_FREETYPE_FONT_RENDER_MODE_BITMAP,
+                                                   LV_TEST_FONT_SIZE,
+                                                   LV_FREETYPE_FONT_STYLE_NORMAL);
+    if(context->font_bitmap == NULL) {
+        LV_LOG_ERROR("freetype font creation failed!");
+    }
+    context->font_outline = lv_freetype_font_create(LV_TEST_FONT_PATH,
+                                                    LV_FREETYPE_FONT_RENDER_MODE_OUTLINE,
+                                                    LV_TEST_FONT_SIZE,
+                                                    LV_FREETYPE_FONT_STYLE_NORMAL);
+    if(context->font_bitmap == NULL) {
+        LV_LOG_ERROR("freetype font creation failed!");
+    }
+#endif
+#if TEST_TINY_TTF
+    context->font_tinyttf = lv_tiny_ttf_create_file("A:" LV_TEST_FONT_PATH, LV_TEST_FONT_SIZE);
+    if(context->font_bitmap == NULL) {
+        LV_LOG_ERROR("freetype font creation failed!");
+    }
+#endif
+    context->scene_act = 0;
+    context->label_perf = lv_label_create(lv_layer_top());
+    lv_obj_set_style_bg_opa(context->label_perf, LV_OPA_COVER, 0);
+    lv_obj_set_style_bg_color(context->label_perf, lv_color_white(), 0);
+    lv_obj_set_style_text_color(context->label_perf, lv_color_black(), 0);
+    lv_obj_set_width(context->label_perf, lv_pct(100));
+#if LV_USE_PERF_MONITOR_LOG_MODE
+    lv_obj_add_flag(context->label_perf, LV_OBJ_FLAG_HIDDEN);
+#endif
+}
+
+static void benchmark_context_deinit(benchmark_context_t * context)
```

Comment:
Suggested name:
```
static void benchmark_context_destroy(benchmark_context_t * context)
```
Release the memory pointed to by the context pointer in this function.